private/buf/cmd/buf/command/registry/label/labelinfo/labelinfo.go

```diff
@@ -39,8 +39,8 @@ func NewCommand(
 ) *appcmd.Command {
 	flags := newFlags()
 	return &appcmd.Command{
-		Use:   name + " <buf.build/owner/repository:label>",
-		Short: "Get label details",
+		Use:   name + " <remote/owner/module:label>",
```

Comment:
The docs should be in the format: `<remote>/<owner>/<module>:<label>`

Reply:
Resolving this since I think we're going to keep it consistent with others for now.